sources/packages/forms/src/form-definitions/sfaa2025-26-ft.json

```diff
@@ -319,28 +319,85 @@
           "label": "Is Change Request Application",
           "type": "hidden",
           "lockKey": true
+        },
+        {
+          "label": "Is program read only",
+          "persistent": false,
+          "calculateValue": "value = (!!data.isReadOnly || data.isChangeRequestApplication);",
+          "calculateServer": true,
+          "key": "isProgramReadOnly",
```

Comment:
`isProgramReadOnly` can cause some confusion between the program itself and the program section.
As a suggestion, we can rename it to:
- `isProgramSectionReadOnly` or
- `isProgramRelatedDateReadOnly`.

Reply:
Got it. I updated to `isProgramSectionReadOnly`